terraform/services/kms-keys/main.tf

```
@@ -0,0 +1,70 @@
+module "standards" {
+  source      = "../../modules/standards"
```

@gsf I suspect there will be changes to the standards module and anywhere this module is referenced by local, relative path could make adopting changes in the future riskier or less appealing. Maybe we go ahead with a git reference on sha starting here?

---

Great idea. @christopher-maboh this should work:


```suggestion
  source      = "github.com/CMSgov/cdap//modules/standards?ref=0bd3eeae6b03cc8883b7dbdee5f04deb33468260"
```

---

Try:
```hcl
  source      = "github.com/CMSgov/cdap//terraform/modules/standards?ref=0bd3eeae6b03cc8883b7dbdee5f04deb33468260"
```